### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by J-C-S-V**
Here is what I did on this milestone:

- Use local storage to keep track of every change on the list
- Add a new JavaScript file (isCompleted.js) and import it as a module.
- Add event listener to the checkbox
- Update items object's value for completed
- Implement a function for the "Clear all completed" button (renderList())

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -12,14 +12,16 @@
   "extends": ["airbnb-base"],
   "rules": {
     "no-shadow": "off",
+    "no-circular": "off",
```
Comment (emmiiorji) [inline .eslintrc.json]: Kindly avoid making changes to the linters configuration files or unnecessary attempts to bypass linter checks. They represent coding standards that you share with your team and most of the time enforce best coding practices.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @J-C-S-V, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a project with a modified linters configuration.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json (hunk 1 above)
Comment (J-C-S-V) [reply .eslintrc.json]: Hello emmiiorji,

Thanks for the heads up, I'll keep that in mind in my next submissions.

Regards.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @J-C-S-V,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**